### PR TITLE
Malaysian road shields.

### PR DESCRIPTION
**Pull request by zerebubuth**
This was sitting on the shelf while we dealt with #1515. I've rebased it to current master.

Connects to #1491.

### CODE REVIEW

File: integration-test/1491-malaysian-road-shields.py [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+        # TODO! should the shield text be '1' or '1-11B1'? not sure from
+        # available sources whether the '-11B1' part is important and/or
+        # usually on signs.
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 68058997,
+                'network': 'MY:federal',
+                'shield_text': '1',
+                'all_networks': ['MY:federal', 'AsianHighway'],
+                'all_shield_texts': ['1', 'AH150'],
```
Comment (nvkelso) [inline integration-test/1491-malaysian-road-shields.py]: `AH150` > `150`?

File: integration-test/1491-malaysian-road-shields.py [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+                'ref': u'MBSA 15',
+                'source': u'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 24701880,
+                'network': u'MY:SGR:municipal',
+                'shield_text': u'MBSA 15',
```
Comment (nvkelso) [inline integration-test/1491-malaysian-road-shields.py]: Drop the space in the `shield_text`?

Comment (nvkelso) [reply integration-test/1491-malaysian-road-shields.py]: Also, this is more than 5 char total so either needs special handling client side or more server transform. Probably use 15 instead?

Comment (nvkelso) [reply integration-test/1491-malaysian-road-shields.py]: From IRL conversation: `BSA15`

File: integration-test/1491-malaysian-road-shields.py (hunk 1 above)
Comment (nvkelso) [reply integration-test/1491-malaysian-road-shields.py]: @nvkelso and @sensescape to research...

Comment (nvkelso) [reply integration-test/1491-malaysian-road-shields.py]: From the Wikipedia page looks like we should really do AsianHighway:CountryCode to make the artwork functional per country:

- https://en.wikipedia.org/wiki/Template:AHN-AH

Comment (nvkelso) [reply integration-test/1491-malaysian-road-shields.py]: Let's just file a followup issue for this and ignore it for this sprint.

**Comment by zerebubuth** [comment]
Re: `BSA15`, fixed in 2b7c81b.

**Pull request merged**